dexs/cables/index.ts

```diff
@@ -0,0 +1,41 @@
+import { FetchOptions, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+
+
+const address: any = {
+  [CHAIN.ARBITRUM]: "0xfA12DCB2e1FD72bD92E8255Db6A781b2c76adC20",
+  [CHAIN.AVAX]: "0xfA12DCB2e1FD72bD92E8255Db6A781b2c76adC20"
+}
+
+const event = "event SwapExecuted(uint256 indexed nonceAndMeta,address taker,address destTrader,uint256 destChainId,address srcAsset,address destAsset,uint256 srcAmount,uint256 destAmount)"
+
+const fetch = async (options: FetchOptions) => {
+  const dailyVolume = options.createBalances();
+  const logs = await options.getLogs({
+    target: address[options.chain],
+    eventAbi: event
+  })
+  logs.forEach(log => {
+    dailyVolume.add(log.destAsset, log.destAmount)
+  })
+  return { dailyVolume }
+};
+
+// it is critical for ARBITRUM RPC to have a fresh timestamp
+const startTimestamp = Math.floor(Date.now() / 1000) - 3 * 60 * 60 * 24;
```

Comment:
no, the start here is for when the protocol started. Used as an indicator to check how far back we should look for daily data